workers/data_refinery_workers/processors/smasher.py

```diff
@@ -130,6 +131,18 @@ def _smash(job_context: Dict) -> Dict:
             experiments[experiment.accession_code] = experiment.to_metadata_dict()
         metadata['experiments'] = experiments
 
+        # Metadata to TSV
+        with open(smash_path + 'metadata.tsv', 'w') as output_file:
+            sample_ids = [k for k in metadata['samples'].keys()]
+            keys = [k for k in metadata['samples'][sample_ids[0]].keys()] + ['sample_id']
```

Comment:
This line is super confusing. I'm not entirely sure what `keys` is a list of after this line except some amount of keys. Any chance you could make this a bit more readable?